src/libs/quotes.ts

```diff
@@ -2,37 +2,52 @@
  * Motivational quotes used in loading screen.
  */
 // eslint-disable-next-line import/prefer-default-export
-export const quotes = [
+const quotesList = [
   {
     author: "Steve Martin",
-    quote: "Be so good, they can't ignore you.",
+    quotes: ["Be so good, they can't ignore you."],
   },
   {
-    quote: "It's hard to beat a person who never gives up.",
     author: "Babe Ruth",
+    quotes: ["It's hard to beat a person who never gives up."],
   },
   {
-    quote:
-      "I've failed over and over and over again in my life. And that is why I succeed.",
     author: "Michael Jordan",
+    quotes: [
+      "I've failed over and over and over again in my life. And that is why I succeed.",
+    ],
   },
   {
-    quote: "Do not let what you cannot do interfere with what you can do.",
     author: "John Wooden",
+    quotes: ["Do not let what you cannot do interfere with what you can do."],
   },
   {
-    quote:
-      "It isn't the mountains ahead to climb that wear you out - it's the pebble in your shoes.",
     author: "Muhammad Ali",
+    quotes: [
+      "It isn't the mountains ahead to climb that wear you out - it's the pebble in your shoes.",
+    ],
   },
   {
-    quote:
-      "We are what we repeatedly do. Therefore, excellence is not an act - but a habit.",
     author: "Socrates",
+    quotes: [
+      "We are what we repeatedly do. Therefore, excellence is not an act - but a habit.",
+    ],
   },
   {
-    quote:
-      "Those who are not courageous enough to take risks will accomplish nothing.",
     author: "Muhammad Ali",
+    quotes: [
+      "Those who are not courageous enough to take risks will accomplish nothing.",
+    ],
   },
 ];
+
```

Comment:
This structure (random author and then random author's quote) is great - a contributor who's a big fan of a particular author can add in a bunch of quotes for them, but it won't cause the unintended side where 90% of the quotes that are served up are from that author. :)